common/lib/xmodule/xmodule/templates/html/image_modal.yaml

```diff
@@ -1,7 +1,23 @@
 ---
 metadata:
-    display_name: Full Screen Image
+    display_name: Full Screen Image Tool
 data: |
-        <h2>Full Screen Image</h2>
-        <p>Use the full screen image tool to allow students to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
-        <a href="http://static.class.stanford.edu/stanford-hills-big.jpg" class="modal-content"><img alt="The Stanford Hills" src="http://static.class.stanford.edu/stanford-hills-small.jpg" /></a>
\ No newline at end of file
+        <h2>Full Screen Image Tool</h2>
+        <p>Use the Full Screen Image tool to allow learners to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
```

Comment:
what does "in the full screen" mean as a definition for this tool? maybe change ...to open the image in the full screen... --> ...to expand the image so that it uses the entire computer screen...
(I'm also not sure what "drag the image within the screen" means if the image is filling the entire screen??)